pkgs/development/python-modules/decora-wifi/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  requests,
+}:
+
+buildPythonPackage rec {
+  version = "1.4";
+  pname = "decora-wifi";
+
+  src = fetchPypi {
+    pname = "decora-wifi";
+    inherit version;
+    hash = "sha256-BIQssxEpwCIyj5z0N2ev9cJr/y907g0Lb7h5iD9dApM=";
+  };
+
+  dependencies = [ requests ];
+
```

Comment:
```suggestion
  pythonImportsCheck = [ "decora_wifi" ];
  
  # No tests in the Pypi source
  doCheck = false
```

Reply:
`doCheck = false` should not be necessary with no hooks to actually do the checking.